src/mon/OSDMonitor.cc

```diff
@@ -12515,7 +12532,7 @@ bool OSDMonitor::prepare_command_impl(MonOpRequestRef op,
 	  goto reply_no_propose;
 	}
       }
-      // fall through
+      break;
```

Comment:
Noticed there was a "break" missing here too; added it for clarity.

Reply:
This actually changes the logic while the removed comment suggests the fallthrough was intentional. Let's bring it back.

Reply:
ACK. That bit is now taken out.